client/src/document.js

```diff
@@ -92,8 +92,11 @@ export class Document extends React.Component {
       return null;
     }
     const translations = [...(doc.other_translations || [])];
-    if (doc.parent) {
-      translations.unshift(doc.parent);
+    if (doc.translation_of) {
```

Comment:
I can't remember why we still have this. Either way, it might be worth saving for another day. 
This kind of computation should probably be done once. In the build process. Otherwise, it has to be done in every browser every single time the React `<Document/>` renders. 
Note though, the `<Document/>` is actually always rendered 1+N times. 1 time in the `ssr` component and then 1 time in the user's browser (the hydration). Either way, if all this file needs to do is just do `const translations = doc.translation || []` it'll save many microseconds but accumulated.